gapic/templates/%namespace/%name_%version/%sub/services/%service/async_client.py.j2

```diff
@@ -54,7 +60,10 @@ from .client import {{ service.client_name }}
 
 {# TODO(yon-mg): handle rest transport async client interaction #}
 class {{ service.async_client_name }}:
-    """{{ service.meta.doc|rst(width=72, indent=4) }}"""
+    """{{ service.meta.doc|rst(width=72, indent=4) }}{% if service.version|length %}
+    This class implements API version {{ service.version }}."""
+    {% else %}"""
```

Comment:
See my comment in the ads template about removing the `else`

Reply:
https://github.com/googleapis/gapic-generator-python/pull/1999/commits/1e6e96e0654f5c941cf569a0d97f866946cc68e3